tsconfig.base.json

```diff
@@ -2,8 +2,8 @@
   "exludes": ["jest.config.base.js"],
   "compilerOptions": {
     "composite": true,
-    "target": "ES2021",
-    "lib": ["ES2021", "DOM"],
+    "target": "ES2022",
+    "lib": ["ES2022", "DOM"],
```

Comment:
I've been experiencing a lot of problems downstream with typescript complaining of the following error:
```
node_modules/@jupiterone/integration-sdk-core/dist/src/errors.d.ts:76:5 - error TS2425: Class 'Error' defines instance member property 'cause', but extended class 'IntegrationError' defines it as instance member function.

76     cause(): Error | undefined;
```
This is because we define `cause` as a method (i.e. `cause(): Error | undefined`) while the parent class in node 18's types defines it as `cause: Error | undefined`.

I've remedied this in the `errors.ts` file. I've also updated the error serializer in `integration-sdk-runtime` using Node's native `inspect`. See their recommendation here:
https://nodejs.org/docs/latest-v18.x/api/errors.html#new-errormessage-options

While it is technically a breaking change **cause**, I don't anticipate major versioning since the only reference I could find to it was in a test in a private package.

Reply:
The switch to ES2022 shouldn't be breaking either as we already required node 18 and it supports all es2022 features.